data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessor.java

```diff
@@ -171,6 +185,17 @@ public Collection<Record<Event>> doExecute(final Collection<Record<Event>> recor
         return records;
     }
 
+    private String transformKey(String key) {
+        if(keyValueProcessorConfig.getTransformKey().equals("lowercase")) {
```

Comment:
Please use static final variable for assigning those magic string (lowercase, uppercase, capitalize, ...)